docs/_docs/guides/013-branching.md

```diff
@@ -149,7 +149,7 @@ running instance.
 `userEmailPattern` is used to lookup the users. Provide the pattern corresponding to your Slack account. `{0}` is the 
 username associated with the branch owner (`-bc` option). 
 
-#### Notifcaitions
+#### Notifications
 
 After a new branch is processed, a Slack notification is sent to the branch owner to inform him that his strings were
```

Comment:
Can we update the part for "him that his strings" with *them that their* as well?

Reply:
nice catch @baktayPin !